src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -36,12 +51,21 @@
 
     private final CaseSearchOperation caseSearchOperation;
     private final ElasticsearchQueryHelper elasticsearchQueryHelper;
+    private final CaseDefinitionRepository caseDefinitionRepository;
+    private final UserRepository userRepository;
+    private final ApplicationParams applicationParams;
 
     @Autowired
     public CaseSearchEndpoint(@Qualifier(AuthorisedCaseSearchOperation.QUALIFIER) CaseSearchOperation caseSearchOperation,
-                              ElasticsearchQueryHelper elasticsearchQueryHelper) {
+                              @Qualifier(CachedCaseDefinitionRepository.QUALIFIER) CaseDefinitionRepository caseDefinitionRepository,
+                              @Qualifier(DefaultUserRepository.QUALIFIER)  UserRepository userRepository,
+                              ElasticsearchQueryHelper elasticsearchQueryHelper,
+                              ApplicationParams applicationParams) {
         this.caseSearchOperation = caseSearchOperation;
         this.elasticsearchQueryHelper = elasticsearchQueryHelper;
+        this.caseDefinitionRepository = caseDefinitionRepository;
+        this.userRepository = userRepository;
+        this.applicationParams = applicationParams;
     }
 
     @PostMapping(value = "/searchCases")
```

Comment:
can you update the Swagger documentation as well with:
@ApiParam(value = "Comma separated list of case type ID(s) or '*' if the search needs to be applied on any existing case type. Note that using '*' is an expensive operation and might have low response times so always prefer explicitly listing the case types when known in advance", required = true)

Reply:
DONE